File: app/src/main/java/org/sabda/gpt/ExamplePrompt.java
package org.sabda.gpt;

import java.util.ArrayList;
import java.util.Random;

public class ExamplePrompt {

    ArrayList<String> prompt;
    Random random;

    public ExamplePrompt(){
        prompt = new ArrayList<>();
        random = new Random();
        initializePrompt();
    }

    private void initializePrompt() {
        prompt.add("Apa itu roti manna?");
        prompt.add("Siapakah nama anak Nuh?");
        prompt.add("Dimana tempat tinggal Ayub?");
        prompt.add("Kapan surat Roma ditulis oleh Paulus?");
        prompt.add("Bagaimana cara Daud mengalahkan Goliat?");
        prompt.add("Mengapa Yudas Iskariot mengkhianati Yesus?");
        prompt.add("Berapa tahun Yesus mulai menjadi pengajar?");
        prompt.add("Beri penjelasan mengenai Filipi 4:6");
        prompt.add("Beri penjelasan mengenai Imannuel");
        prompt.add("Sebutkan nama murid Tuhan Yesus");
        prompt.add("Sebutkan sifat-sifat Allah");
        prompt.add("Penggalian studi kata \"Kasih dan Pengharapan\" menurut bahasa asli");
    }

    public String getRandomPrompt(){
        int randomIndex = random.nextInt(prompt.size());
        return prompt.get(randomIndex);
    }
}

File: androidApp/src/main/java/org/sabda/gpt/data/local/ExamplePrompt.kt
package org.sabda.gpt.data.local

import java.util.Random

class ExamplePrompt {

    private val prompts = listOf(
        "Apa itu roti manna?",
        "Siapakah nama anak Nuh?",
        "Dimana tempat tinggal Ayub?",
        "Kapan surat Roma ditulis oleh Paulus?",
        "Bagaimana cara Daud mengalahkan Goliat?",
        "Mengapa Yudas Iskariot mengkhianati Yesus?",
        "Berapa tahun Yesus mulai menjadi pengajar?",
        "Beri penjelasan mengenai Filipi 4:6",
        "Beri penjelasan mengenai Imannuel",
        "Sebutkan nama murid Tuhan Yesus",
        "Sebutkan sifat-sifat Allah",
        "Penggalian studi kata \"Kasih dan Pengharapan\" menurut bahasa asli"
    )

    private val random = Random()

    fun getRandomPrompt(): String {
        return prompts[random.nextInt(prompts.size)]
    }
}